packages/zapp/console/src/components/Executions/TaskExecutionsList/MapTaskExecutionDetails.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { MapTaskExecution } from 'models/Execution/types';
+import { TaskExecutionPhase } from 'models/Execution/enums';
+import { formatRetryAttempt, getTaskRetryAtemptsForIndex } from './utils';
+import { TaskExecutionLogsCard } from './TaskExecutionLogsCard';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  wrapper: {
+    padding: `${theme.spacing(2)}px ${theme.spacing(3)}px`,
```

Comment:
nit: can be used as `theme.spacing(2, 3)` instead. No need to change, just for future reference :)

Reply:
will add to the next PR, copy-paste from other place 🙃